src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java

```diff
@@ -158,6 +159,8 @@ private Node getEntryNode(BibEntry entry) {
     private IconTheme.JabRefIcons getIcon(EntryType type) {
         if (StandardEntryType.Book.equals(type)) {
             return IconTheme.JabRefIcons.BOOK;
+        } else if (Arrays.asList(StandardEntryType.InBook, StandardEntryType.InProceedings, StandardEntryType.InCollection).contains(type)) {
```

Comment:
As tobiasdiez converted the stuff to Enums (I just have not the code at hand, but should be) , it is more efficient to use an EnumSet.
And for enums you can also replace equals with ==